tests/functional_tests/iterator_tests/test_builtins.py

```diff
@@ -141,6 +166,26 @@ def test_arithmetic_and_logical_builtins(program_processor, builtin, inputs, exp
         assert np.allclose(np.asarray(out), expected)
 
 
+@pytest.mark.parametrize("builtin, inputs, expected", arithmetic_and_logical_test_data())
+def test_unapplied_arithmetic_and_logical_builtins_gtfn(builtin, inputs, expected):
```

Comment:
Unapplied sounds a bit weird to me:
```suggestion
def test_arithmetic_and_logical_functors_gtfn(builtin, inputs, expected):
```

Reply:
I like version more, but not enough to start a fight. ;)